src/_js/main.mjs

```diff
@@ -91,4 +106,6 @@ ga.q = [];
 ga('create', UA_ID, 'auto');
 ga('set', 'referrer', document.referrer.split('?')[0]);
 ga('send', 'pageview');
-insertScript('https://www.google-analytics.com/analytics.js');
+const script = document.createElement('script');
```

Comment:
Why was this changed back / it seems to work fine when trying to `import`?

Reply:
Oh, right, they don't send CORS headers for localhost, but they do for anything else. Changing back to dynamic import then.